client/src/page-not-found/index.tsx

```diff
@@ -22,24 +15,16 @@ const LANGUAGES = new Map(
 
 export function PageNotFound() {
   const location = useLocation();
-  const [url, setURL] = useState("");
+  const [url, setURL] = React.useState("");
```

Comment:
Curious, is this a pattern we should adopt across the codebase?

Reply:
The best thing about using [getters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) and [setters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/set) in JavaScript is that you can keep both (which are basically the same property) lowercase key.
But this has nothing to do with the React state unfortunately, I should understand if we can turn around this 🤔

```JavaScript
class NotReactComponent {
 get url(){}
 set url(url){}
}
```

Reply:
> Curious, is this a pattern we should adopt across the codebase?

Yes please.